src/main/java/br/com/sbk/sbking/networking/server/SBKingServer.java

```diff
@@ -0,0 +1,173 @@
+package br.com.sbk.sbking.networking.server;
+
+import static br.com.sbk.sbking.logging.SBKingLogger.LOGGER;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import br.com.sbk.sbking.core.Board;
+import br.com.sbk.sbking.core.Card;
+import br.com.sbk.sbking.core.Deal;
+import br.com.sbk.sbking.core.Direction;
+import br.com.sbk.sbking.core.Player;
+import br.com.sbk.sbking.core.rulesets.RulesetFromShortDescriptionIdentifier;
+import br.com.sbk.sbking.core.rulesets.abstractClasses.Ruleset;
+import br.com.sbk.sbking.gui.models.PositiveOrNegative;
+import br.com.sbk.sbking.networking.kryonet.IdentifierToPlayerMap;
+import br.com.sbk.sbking.networking.kryonet.KryonetSBKingServer;
+import br.com.sbk.sbking.networking.server.gameServer.GameServer;
+import br.com.sbk.sbking.networking.server.gameServer.KingGameServer;
+import br.com.sbk.sbking.networking.server.gameServer.MinibridgeGameServer;
+
+/**
+ * This class has two responsibilities: 1: receiving method calls from the
+ * GameServer and sending to client(s) over the network layer
+ * (KryonetSBKingServer). 2: receiving method calls from the network layer and
+ * act on the GameServer (via Table).
+ */
+public class SBKingServer {
+
+  IdentifierToPlayerMap identifierToPlayerMap = new IdentifierToPlayerMap();
```

Comment:
Here's a style question: do you prefer to initialize your variables inside your constructors? If yes, then we could move this to the SBKingServer constructor on line 36 I think

Reply:
Good comment. I think I do, but never really thought about it.
The visibility is certainly better and I will fix here and check if that could be a checkstyle rule. I would gladly put it as an error :)